modules/ROOT/pages/dataweave-cookbook-insert-attribute.adoc

```diff
@@ -4,37 +4,20 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: studio, anypoint, transform, transformer, format, xml, json, metadata, dataweave, data weave, datamapper, dwl, dfl, dw, output structure, input structure, map, mapping
 
-This DataWeave example uses `@(key:value)` syntax (for example, `@(loc: "US")`) to inject attributes to XML tags. The example populates the "title" tag with two attributes, and each of the "author" tags with one too.
+This DataWeave example uses `@(key:value)` syntax (for example, `@(loc: "US")`) to inject attributes to XML tags. The example populates the `title` element with two attributes and each of the `author` elements with one attribute.
 //LINK TO DW 1.0 EXAMPLES:
 include::partial$dataweave1-links.adoc[tag=dataweave1Examples]
 
 It uses these functions:
 
-* `mapObject` to go through each of the books in the input.
-* `map` to go through the authors in each book.
+* `mapObject` to iterate over each of the books in the input XML, defined in the variable `myInput`.
+* `map` to iterate over the `author` elements in each book.
```

Comment:
I had to look this up, but you do not need to follow "iterate" with any kind of preposition:
"...iterate each of the books...."
"...iterate the `author` elements...."

Reply:
"iterate over" or "iterate through" is the most common way this is expressed.